cli/Makefile

```diff
@@ -28,13 +28,23 @@ endif
 .PHONY: build
 build: export CGO_ENABLED=0
 build:
-	gox -ldflags '-X github.com/hasura/graphql-engine/cli/version.BuildVersion=$(VERSION) -X github.com/hasura/graphql-engine/cli/plugins.IndexBranchRef=$(PLUGINS_BRANCH) -s -w -extldflags "-static"' \
+	gox -ldflags '-X github.com/hasura/graphql-engine/cli/version.BuildVersion=$(VERSION) -X github.com/hasura/graphql-engine/cli/plugins.IndexBranchRef=$(PLUGINS_BRANCH) -X github.com/hasura/graphql-engine/cli/commands.IsUpdateCommandEnabled=true -s -w -extldflags "-static"' \
 	-rebuild \
 	-os="$(OS)" \
 	-arch="amd64" \
 	-output="$(OUTPUT_DIR)/$(VERSION)/cli-hasura-{{.OS}}-{{.Arch}}" \
 	./cmd/hasura/
 
+.PHONY: build-brew
```

Comment:
I feel like we don't need an entry in `Makefile` entry since the brew formula directly sets the [ld flags](https://github.com/leos/homebrew-core/blob/c675cc5efb6315de4f7b8f22cc76ce6a70a3cf55/Formula/hasura-cli.rb#L12)

Reply:
Right, okay. I shall remove this entry. But the formula does not include the flag `IsUpdateCommandEnabled`. Should it be updated in the formula?

Reply:
Cool, I think the formula change can be done later

Reply:
Have removed this.